Source/Assets/PlayFabEditorExtensions/Editor/Scripts/Panels/PlayFabEditorSDKTools.cs

```diff
@@ -202,11 +202,11 @@ private static void ShowSdkNotInstalledMenu()
                     var buttonWidth = 150;
 
                     GUILayout.FlexibleSpace();
-                    if (GUILayout.Button("Install PlayFab SDK", PlayFabEditorHelper.uiStyle.GetStyle("Button"), GUILayout.MaxWidth(buttonWidth),
-                        GUILayout.MinHeight(32)))
-                    {
+                    if (GUILayout.Button("Refresh", PlayFabEditorHelper.uiStyle.GetStyle("Button"), GUILayout.MaxWidth(buttonWidth), GUILayout.MinHeight(32)))
```

Comment:
since that 32 is hard coded twice, it may be more readable to give it a const with a name.

Reply:
Agree. Let's fix it next time.